client/public/index.html

```diff
@@ -13,7 +13,9 @@
 
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/snap.svg/0.5.1/snap.svg.js">
```

Comment:
Wait, why do you need snap? Your manipulation isn't that tricky here, is it?

Reply:
yea i don't; forgot to remove that when i scrapped the idea of using snap 😅